lib/commands/ls.js

```diff
@@ -23,5 +23,6 @@ class LsCommand extends Command {
 
 LsCommand.description = 'View running ghost processes';
 LsCommand.global = true;
+LsCommand.aliases = ['status'];
```

Comment:
No, status should be a valid object key w/o quotes, not sure why Vim would be highlighting it.